src/erlcloud_iam.erl

```diff
@@ -900,7 +934,20 @@ data_type("UserDetail") ->
 data_type("UserPolicyList") ->
     [{"PolicyDocument", policy_document, "Uri"},
      {"UserName", user_name, "String"},
-     {"PolicyName", policy_name, "String"}].
+     {"PolicyName", policy_name, "String"}];
+data_type("EvaluationResults") ->
+    [{"EvaluationResults/member",
+      evaluation_result_list, data_type("EvaluationResult")},
+     {"Marker", marker, "String"},
```

Comment:
I do not think these two (`marker`|`is_truncated`) are needed given that API uses `iam_query_all` and does not provide raw marker api.